doc/get_started_on_kind.md

```diff
@@ -0,0 +1,35 @@
+# Get started on kind
+
+This document describes how to deploy Chaos Mesh in Kubernetes on your laptop (Linux or macOS) using kind.
+
+## Prerequisites
+
+Before deploying Chaos Mesh, make sure the following items have been installed. 
+
+- [Docker](https://docs.docker.com/install/) (required when running in [kind](https://kind.sigs.k8s.io/))
```

Comment:
If there is only 1 item, you don't have to use a list.  Just combine the sentence into one paragraph.